core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -1167,6 +1160,21 @@ class LogManager(logDirs: Seq[File],
 
 object LogManager {
 
+  /**
+   * wait all jobs to complete
+   * @param jobs jobs
+   * @param callback handle the exception caused by Future#get
```

Comment:
We should mention that this will be called for each future that throws an exception.